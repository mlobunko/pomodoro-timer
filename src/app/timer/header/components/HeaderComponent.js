import React from "react";

import { IconInfo } from "../icons";
import { StyledHeader, StyledTomatoIcon, StyledInfo } from "./styles";

export const HeaderComponent = () => (
  <StyledHeader>
    P<StyledTomatoIcon />
    omodoro Timer
    <StyledInfo>
      <a
        href="https://en.wikipedia.org/wiki/Pomodoro_Technique"
        target="_blank"
        rel="noopener noreferrer"
        aria-label="wiki link on article about Pomodoro Technique"
      >
        <IconInfo />
      </a>
    </StyledInfo>
  </StyledHeader>
);

HeaderComponent.displayName = "Header";

export default HeaderComponent;
