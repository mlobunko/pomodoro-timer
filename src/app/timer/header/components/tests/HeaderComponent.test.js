import React from "react";
import { shallow } from "enzyme";

import { HeaderComponent } from "../HeaderComponent";

describe("render", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<HeaderComponent />);
    expect(wrapper).toMatchSnapshot();
  });
});
