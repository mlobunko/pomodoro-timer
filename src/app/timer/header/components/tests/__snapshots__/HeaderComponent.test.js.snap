// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render renders correctly 1`] = `
<StyledHeader>
  P
  <StyledTomatoIcon />
  omodoro Timer
  <StyledInfo>
    <a
      aria-label="wiki link on article about Pomodoro Technique"
      href="https://en.wikipedia.org/wiki/Pomodoro_Technique"
      rel="noopener noreferrer"
      target="_blank"
    >
      <IconInfo />
    </a>
  </StyledInfo>
</StyledHeader>
`;
